includes/admin.form.inc

```diff
@@ -14,5 +14,27 @@ function islandora_marcxml_admin_settings(array $form, array &$form_state) {
     '#description' => t("This provides an extra option to be available under the 'edit' link present on objects that have MODS datastreams."),
     '#default_value' => variable_get('islandora_marcxml_allow_editing_of_existing_mods', FALSE),
   );
+  $form['islandora_marcxml_select_marc_to_mods_xsl'] = array(
+    '#type' => 'select',
+    '#title' => t('MARC to MODS XSL file'),
+    '#default_value' => 'MARC21slim2MODS3-6.xsl',
```

Comment:
Shouldn't this be
`    '#default_value' => variable_get('islandora_marcxml_select_marc_to_mods_xsl', 'MARC21slim2MODS3-6.xsl'),
`